src/topp/FileConverter.cpp

```diff
@@ -446,7 +446,7 @@ class TOPPFileConverter :
                                                  CONVERSION_MZXML));
       MzXMLFile f;
       f.setLogType(log_type_);
-      ChromatogramTools().convertChromatogramsToSpectra<MSExperimentType>(exp);
+      //ChromatogramTools().convertChromatogramsToSpectra<MSExperimentType>(exp);
```

Comment:
this some very old legacy change (by Andreas if I remember correctly).
It basically converts a chromatogram into n separate ms/ms spectra and attaches them to the experiment. It (rightfully) confuses MaxQuant and I cannot see a good reason why we should do this.